Source/WebCore/svg/SVGTransformable.cpp

```diff
@@ -99,67 +99,126 @@ template<typename CharacterType> static int parseTransformParamList(StringParsin
 static constexpr std::array requiredValuesForType { 0, 6, 1, 1, 1, 1, 1 };
 static constexpr std::array optionalValuesForType { 0, 0, 1, 1, 2, 0, 0 };
 
-template<typename CharacterType> static std::optional<SVGTransformValue> parseTransformValueGeneric(SVGTransformValue::SVGTransformType type, StringParsingBuffer<CharacterType>& buffer)
+template<typename CharacterType> static RefPtr<SVGTransform> parseTransformGeneric(SVGTransformValue::SVGTransformType type, StringParsingBuffer<CharacterType>& buffer, RefPtr<SVGTransform> reusableValue)
```

Comment:
Should we use the same name the caller passes above? reusableValue -> existingTransform?

Or at least do not use Value in the name to not confuse it with SVGTransformValue. reusableValue -> reusableTransform?